src/components/BingoBoardCard.vue

```diff
@@ -90,27 +113,33 @@ function reset() {
     }
   }
   clearInterval(startBingo.value);
-  show.value = false;
+  showBingo.value = false;
   bingoNumbers.value = newNumbers;
+  includedColumnIndexes.value = [];
 }
 reset();
+
+function getColumnStyle(index) {

```

Comment:
add comments on function with what is the use of the function

Reply:
okay